scripts/claude-github-actions-check.js

```
@@ -0,0 +1,252 @@
+    const isHealthy = checkBuildMonitoring();
+    checkMissingRuns();
+    console.log(`\n🏥 Overall Status: ${isHealthy ? '✅ HEALTHY' : '⚠️  NEEDS ATTENTION'}`);
+    break;
+    
+  case 'activity':
+    showRecentActivity(detailed);
+    break;
+    
+  case 'logs':
+    const logFile = saveLogs();
+    console.log('\n📋 Recent activity with full logs saved for Claude analysis');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)